docs/user/PangenomeAnalyses/pangenomeStat.md

```diff
@@ -144,10 +144,19 @@ To generate these files, use the `write_pangenome` subcommand with the `--partit
 `ppanggolin write_pangenome -p pangenome.h5 --partitions`
 
 
-#### Gene Families to Genes Associations
 
-The `gene_families.tsv` file mirrors the format provided by [MMseqs2](https://github.com/soedinglab/MMseqs2) through its `createtsv` subcommand. This file structure comprises three columns: the gene family name in the first column, the gene names in the second, and a third column that remains empty or contains an "F" to denote potential gene fragments instead of complete genes. This indication appears only if the [defragmentation](./pangenomeCluster.md#defragmentation) pipeline has been used.
+#### Gene Families to Gene Associations
+
+The `gene_families.tsv` file follows the format produced by [MMseqs2](https://github.com/soedinglab/MMseqs2) using the `createtsv` subcommand. The file consists of four columns:
```

Comment:
This is technically not correct since we added columns. Do we still want to mention this in the doc?

Reply:
I remove this part.